_sass/_layout.scss

```diff
@@ -372,3 +373,31 @@ q:after {
 .downloads-content p:first-child{
   color: red;
 }
+
+
+/**
+  * Style for posts pages
+*/
+.post-content{
```

Comment:
This could go to a new scss for files. Don't forget about adding the final empty line.